datascience/tables.py

```diff
@@ -514,7 +514,174 @@ def last(self, label):
     ############
 
     def set_format(self, column_or_columns, formatter):
-        """Set the format of a column."""
+        """
```

Comment:
Could you please add an example of one of the built-in formatter classes near the top of this docstring? Most of the users of our library, I image, would be using these quite frequently.